packages/chat/src/ChatClient.ts

```diff
@@ -831,20 +835,25 @@ export class ChatClient extends IrcClient {
 			}
 		});
 
-		this.onTypedMessage(ClearChat, ({ params: { channel, user }, tags }) => {
+		this.onTypedMessage(ClearChat, clearChat => {
```

Comment:
The message objects are generally called `msg`.

Reply:
My bad, thought I could name it [like this](https://github.com/twurple/twurple/blob/975f90af3e49dc80336c178ad2796a38478ed63d/packages/chat/src/ChatClient.ts#L917) as well